lisa/features/disks.py

```diff
@@ -64,6 +64,15 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
     def get_resource_disk_mount_point(self) -> str:
         raise NotImplementedError
 
+    # Get disk controller type of the VM by checking the OS disk partition
+    @property
+    def controller_type(self) -> str:
+        os_disk = self.get_partition_with_mount_point("/")
+        if os_disk.disk == 'nvme':  # noqa BLK100
```

Comment:
remove  # noqa BLK100

Reply:
Fixed this by changing the logic to verify disc controller type.